ui/app/styles/core/tag.scss

```diff
@@ -22,7 +23,7 @@
   vertical-align: middle;
 
   code {
-    color: $grey;
+    color: var(--token-color-foreground-primary);
```

Comment:
I'm curious if there are any variables we can remove from `_color_variables.scss` after replacing all of these

Reply:
I didn't remove all of them -- only the ones that were causing test failures. But I'll check if any of the ones I updated are completely replaced